src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -243,4 +260,9 @@ public void writeThrough(Activity activity) {
     activityEntity.setProperty("display_string_parameters", activity.getDisplayStringParameters());
     datastore.put(activityEntity);
   }
+  public void writeThrough(Setting setting){
```

Comment:
same here. maybe dont use background-color.

Reply:
got it.